Feature share html feedback

manuelplazaspalacio opened the pull request:
Close #1568

idelcano committed:
working on html intent

manuelplazaspalacio committed:
Sending attached html.

manuelplazaspalacio committed:
Merge branch 'v1.2_hnqis' into feature-share_html_feedback

# Conflicts:
#	app/src/main/java/org/eyeseetea/malariacare/fragments/PlanActionFragment.java

manuelplazaspalacio committed:
Solving merge bugs

xurxodev committed:
Merge branch 'v1.2_hnqis' into feature-share_html_feedback

# Conflicts:
#	app/src/main/java/org/eyeseetea/malariacare/fragments/PlanActionFragment.java

xurxodev reviewed (changes requested):
@manuelplazaspalacio I have found two problems:
- the design of html is not equal to http://web2.beckolsystem.com/hnqis/#/screens/6dc87837-756f-4d8a-a188-c06f9345e4be
- In this branch empty parents without critical questions answered badly are again visible, please check the branch development (v1.2_hnqis) because there it does well

manuelplazaspalacio committed:
Solve bugs with styles:

-Add inline styles for work when open directly from gmail.
-First paragraph match the size of the image.
-Add charset utf-8 to work well with special characters.

manuelplazaspalacio committed:
Merge branch 'v1.2_hnqis' into feature-share_html_feedback

manuelplazaspalacio committed:
Merge remote-tracking branch 'origin/feature-share_html_feedback' into feature-share_html_feedback

# Conflicts:
#	app/src/main/java/org/eyeseetea/malariacare/fragments/PlanActionFragment.java

manuelplazaspalacio commented:
Changes done @xurxodev

xurxodev reviewed (approved):
thanks @manuelplazaspalacio
